news/templates/news/user_news.html

```diff
@@ -1,47 +1,41 @@
 {% load i18n %}
 {% load news %}
 
-{% for item in alerts %}
-<div class="alert alert-{{ item.alert }}">
-  <h4>
-    {{ item.title|safe }}
-    {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears" %} {{ item.publish }}</span></small>{% endif %}
-    {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
-  </h4>
-  <div>
-    {{ item.body|safe }}
-  </div>
-</div>
-{% endfor %}
 
 {% if news %}
-<div class="panel panel-default news-panel">
-  <div class="panel-heading">
-    <h4 class="panel-title">{% trans "Course news" %}</h4>
-  </div>
-  <div class="list-group">
-    {% for item in news %}
-    <a class="list-group-item{% if item.collapsed %} collapsed{% endif %}" href="#news{{ item.id }}" data-toggle="collapse" role="button" aria-controls="news{{ item.id }}" aria-expanded="{% if item.collapsed %}false{% else %}true{% endif %}">
-      <h4 class="list-group-item-heading">
-        <span class="caret"></span>
-        {{ item.title|safe }} <small>{{ item.publish }}</small>
-        {% if not item|is_published:now %}<small><span class="label label-default">{% trans "Appears later" %}</span></small>{% endif %}
-        {% if is_course_staff %}<small><span class="label label-default">{{ item.audience|news_audience }}</span></small>{% endif %}
-      </h4>
-    </a>
-    <div class="list-group-item-text collapse{% if not item.collapsed %} in{% endif %}" id="news{{ item.id }}">
-      {{ item.body|safe }}
-    </div>
-    {% if item.begin_more %}
-    <a class="list-group-item" href="#more-news" data-toggle="collapse" role="button" aria-controls="more-news" aria-expanded="false">
-      {% trans "Show more" %}
-    </a>
-    <div class="collapse" id="more-news">
-    {% endif %}
-    {% endfor %}
-    {% if more > 0 and news|length > more %}
-    </div>
-    {% endif %}
-  </div>
-</div>
+	<div class="panel panel-default news-panel">
+		<div class="panel-heading">
+			<h3 class="panel-title">{% trans "Course news" %}</h3>
+		</div>
+		<div class="list-group">
+			{% for item in news %}
+				<div class="{% if item.pin %}pinned-{% endif %}list-group-item">
+					<h4 class="list-group-item-heading">
+						{{ item.title|safe }}
+						{% if not item|is_published:now %}
+							<small class="future-instance">{% trans "Published on" %}: {{ item.publish }}</small>
+						{% else %}
+							<small class="current-instance">{{ item.publish }}</small>
+						{% endif %}
+						{% if is_course_staff %}
+							<small><span class="label label-primary">{{ item.audience|news_audience }}</span></small>
+						{% endif %}
+						{% if item.pin %}
+							<small><span class="glyphicon glyphicon-pushpin"></span></small>
```

Comment:
Since this is the top (and implied, most recent) item, I'm concerned that there's no way for screen reader users to know it's pinned. I can imagine someone listening to the title/date of the pinned item, and thinking "I've seen this before and it's the newest, so I can skip past the entire news block".

How about adding a screen reader specific note alongside the pin icon?

Reply:
I added a sr-only-comment there, actually already before the date, so that it should be clear that the item is pinned. I have not added those sr-onlys before, so if there is something off with it, please let me know :)